rllib/utils/test_utils.py

```diff
@@ -1261,3 +1266,323 @@ def check(self, rtol=None):
         # same input and all nets have the same (dummy) weight values.
         for v in self.output_values.values():
             check(v, self.output_values[main_key], rtol=rtol or 0.002)
+
+
+def get_mean_action_from_algorithm(alg: "Algorithm", obs: np.ndarray) -> np.ndarray:
```

Comment:
let's make this method private? I don't think it has any other utility outside of this module

Reply:
done